branca/element.py

```diff
@@ -551,14 +551,15 @@ def render(self, **kwargs):
             iframe = (
             '<div style="width:{width};">'
             '<div style="position:relative;width:100%;height:0;padding-bottom:{ratio};">'  # noqa
-            '<iframe src="{html}" style="position:absolute;width:100%;height:100%;left:0;top:0;">'  # noqa
+            '<iframe src="{html}" style="position:absolute;width:100%;height:100%;left:0;top:0;'  # noqa
+            'border:none !important;">'
```

Comment:
No need to bread the line here. This reads better in a single line and we already have the `noqa` there.